src/client/utils/browser.js

```diff
@@ -22,40 +22,56 @@ function calculateBrowserAndVersion (userAgent) {
     };
 }
 
+
+export var isIE11;
```

Comment:
1)remove from browser.js `init();`
2)add to Hammerhead constructor browserUtil.init(window)

browserUtils doesn't initialized on `require`